bin/run_ci_dataset.sh

```diff
@@ -90,11 +87,9 @@ fi
 max_proc=8
 NUMPROC=${NUMPROC:-$((sys_proc < max_proc ? sys_proc : max_proc))}
 
-echo "Running ap_verify on ${DATASET} ${GEN}..."
+echo "Running ap_verify on ${DATASET}..."
 ap_verify.py --dataset "${DATASET}" \
-    ${GEN} \
     ${PIPE} \
     --output "${WORKSPACE}" \
     --processes "${NUMPROC}" \
-    --metrics-file "${WORKSPACE}/ap_verify.{dataId}.verify.json" \
```

Comment:
Why did this line also go?

Reply:
This argument was Gen 2-only, and was removed on c145de5eb67490e1df7a5708450e7de6a9b9777f.